### PR TITLE
Rename fileList selector

### PR DESCRIPTION
**Pull request by PVince81**
For https://github.com/nextcloud/server/pull/33320

### CODE REVIEW

File: .github/workflows/cypress.yml
Comment (artonge) [inline .github/workflows/cypress.yml:30]: ?

**Review by artonge — APPROVED** [review]
Looks ready to merge once parent PR is merged

File: .github/workflows/cypress.yml
Comment (PVince81) [reply .github/workflows/cypress.yml:30]: this makes CI run against the branch from https://github.com/nextcloud/server/pull/33320

once https://github.com/nextcloud/server/pull/33320 is merged I'll revert this back

**Comment by PVince81** [comment]
all green

**Pull request merged**
